app/src/main/res/values-ru/strings.xml

```diff
@@ -1,8 +1,8 @@
-<resources xmlns:tools="http://schemas.android.com/tools">
+<resources tools:ignore="ExtraTranslation" xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Extra translations must be removed and not ignored. Please remove this ignore tag.

Reply:
Ready